syntaxes/openfga.injection.json

```diff
@@ -8,7 +8,7 @@
   ],
   "repository": {
     "openfga-block-scalar": {
-      "begin": "(?i)(^model)(:)\\s+(\\|)\\s*\\n",
+      "begin": "(?i)(^model)(:)\\s+(\\|)(\\+)?\\s*\\n",
```

Comment:
Should we just go ahead and support both of the operators while we're here? There's `+` (keep) and `-` (strip)

https://yaml.org/spec/1.2-old/spec.html#id2794534